edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,148 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Awaiting Proctoring Software Installation and Setup
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      You have chosen to take {{display_name}} as a proctored exam. You should be directed to a new window
+      with installation and setup instructions. You can also <a href="{{software_download_url}}" target="_blank">open the installation window directly.</a>
+    {% endblocktrans %}
+  </p>
+  <div class="proctored-exam-message">
+    <h3>
+    {% blocktrans %}
+      Here is your unique exam code. You will be asked for it during the setup.
+    {% endblocktrans %}
+    </h3>
+    <h2> {{exam_code}}<span class='copy-to-clipboard'></span></h2>
+    <p>
+      {% blocktrans %}
+        Do not share this code. It is linked to your {{platform_name}} account and can be used only once.
+      {% endblocktrans %}
+    </p>
+  </div>
+</div>
+<div class="footer-sequence border-b-0 padding-b-0">
+  <span>
+    {% blocktrans %}
+      Note: As soon as you finish installing and setting up the proctoring software,
+      you will be prompted to start your timed exam.
+    {% endblocktrans %}
+   </span>
+    <p>
+    {% blocktrans %}
+      Be prepared to start your exam and to complete it while adhering to the {{platform_name}}
+      rules for online proctoring.
```

Comment:
"Be prepared to start your exam and complete it while adhering to edX's Online Proctoring Rules." 
@chrisndodge If it's possible, can we make "Online Proctoring Rules" link text with the target: http://edx.readthedocs.org/projects/edx-guide-for-students/en/latest/sfd_proctoredexams/SFD_OnlineProctoringPolicy.html